src/onnx/onnx.cpp

```diff
@@ -2105,6 +2106,47 @@ struct onnx_parser
         return l0;
     }
 
+    enum reduce_mode_t
```

Comment:
This should be `enum class` to avoid name collision.

Reply:
fixed